proto/protocolapis/astria/protocol/bridge/v1alpha1/types.proto

```diff
@@ -2,9 +2,19 @@ syntax = "proto3";
 
 package astria.protocol.bridge.v1alpha1;
 
+import "astria/primitive/v1/types.proto";
+
 // A response containing the last tx hash given some bridge address,
 // if it exists.
 message BridgeAccountLastTxHashResponse {
   uint64 height = 2;
   optional bytes tx_hash = 3;
 }
+
+message BridgeAccountInfoResponse {
+  uint64 height = 2;
+  astria.primitive.v1.RollupId rollup_id = 3;
+  optional bytes asset_id = 4;
```

Comment:
What does it mean if this field is not set?

Also, I wonder if we should start using IBC denoms (i.e. strings) in their place - so this would be `ibc/<hash>`. I think it would make things a bit neater.

Reply:
it means the account is not a bridge account - will add a comment to clarify